src/main/java/net/cytonic/cytosis/commands/moderation/UnbanCommand.java

```diff
@@ -44,7 +44,7 @@ public UnbanCommand() {
                 return;
             }
             Cytosis.getDatabaseManager().getMysqlDatabase().unbanPlayer(uuid,new Entry(uuid, actor.getUuid(), Category.UNBAN, "command"));
-            sender.sendMessage(MM."<GREEN><b>UNBANNED!</green></b><gray>\{player} was successfully unbanned!");
+            sender.sendMessage(MM."<GREEN><b>UNBANNED!</green><gray>\{player} was successfully unbanned!");
```

Comment:
You just have it nested improperly. Also you need a space between players name and the unbanned.
Correct:
`sender.sendMessage(MM."<green><b>UNBANNED!</b></green><gray> \{player} was successfully unbanned!");`